addons/portal/wizard/portal_wizard_views.xml

```diff
@@ -31,7 +31,7 @@
                     <field name="welcome_message"
                         placeholder="This text is included at the end of the email sent to new portal users."
                         class="mb-3"/>
-                    <field name="user_ids" class="o_portal_wizard_user_ids">
+                    <field name="user_ids" class="o_portal_wizard_user_ids" widget="portal_wizard_user_one2many">
```

Comment:
To check as discussed: isn't it a way to avoid closing the wizard with a True/False/bisous returned value ?